testing/mod.ts

```diff
@@ -106,7 +106,7 @@ function printResults(
   }
   // Attempting to match the output of Rust's test runner.
   console.log(
-    `\ntest result: ${stats.failed ? RED_FAILED : GREEN_OK}. ` +
+    `\ntest result: ${stats.failed ? RED_FAILED : TRIM_GREEN_OK}. ` +
```

Comment:
@zekth can you replace both variables with explicit color calls? Seems superfluous to have another variable just for this

Reply:
yes refactoring this : https://github.com/denoland/deno_std/pull/314/files/311dae3af0c9660cdc10cd24063debc273ff76e9#diff-21d94e5c829319e0d4c4acd5e1cc2bd3R45